src/EditorFeatures/Core.Wpf/InlineErrors/InlineErrorFormatDefinition.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable disable
+
+using System;
+using System.ComponentModel.Composition;
+using System.Windows.Media;
+using Microsoft.CodeAnalysis.Editor.Implementation.EditAndContinue;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.VisualStudio.Language.StandardClassification;
+using Microsoft.VisualStudio.Text.Adornments;
+using Microsoft.VisualStudio.Text.Classification;
+using Microsoft.VisualStudio.Utilities;
+
+namespace Microsoft.CodeAnalysis.Editor.InlineErrors
+{
+    internal sealed class ClassificationTypeDefinitions
+    {
+        [Export]
+        [Name("IE: " + PredefinedErrorTypeNames.SyntaxError)]
```

Comment:
IE?  please don't abbr.

Reply:
stands for inline errors, should I do something else for the naming?